draft-ietf-quic-transport.md

```diff
@@ -646,10 +661,10 @@ that the client selected.
 
 When the client receives a Version Negotiation packet from the server, it should
 select an acceptable protocol version.  If the server lists an acceptable
-version, the client selects that version and resends all packets using that
-version. The resent packets MUST use new packet numbers.  These packets MUST
-continue to have the VERSION flag set and MUST include the new negotiated
-protocol version.
+version, the client selects that version and reattempts to created a connection
+using that version.  The client MUST update packet numbers when attempting to
+connect with the updated version.  Packets MUST continue to have the VERSION
```

Comment:
I think "The client MUST update packet numbers when attempting to continue to connect with the updated version." is unclear.  How about  "The client must select a new initial random packet number.", since you mention including the version in the next sentence?